### PR TITLE
Add libtorrent and rtorrent.

### PR DESCRIPTION
**Pull request by jedavies-dev**
## Description of package
rtorrent: A commandline BitTorrent client.
libtorrent: Required by rtorrent

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/libtorrent/build [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr
+
+make
+make DESTDIR="$1" install
+
```
Comment (dylanaraps) [inline community/libtorrent/build]: Remove this line.

File: community/libtorrent/sources [hunk 2]
```diff
@@ -0,0 +1 @@
+http://rtorrent.net/downloads/libtorrent-0.13.8.tar.gz
```
Comment (dylanaraps) [inline community/libtorrent/sources]: No https source available?

File: community/libtorrent/build (hunk 1 above)
Comment (jedavies-dev) [reply community/libtorrent/build]: Removed  the empty line

File: community/libtorrent/sources (hunk 2 above)
Comment (jedavies-dev) [reply community/libtorrent/sources]: Have changed to use https.  For some reason they use a http link on their download page.

**Comment by dylanaraps** [comment]
:+1:

**Pull request merged**